docs/builder/build_customized_backend.md

```diff
@@ -0,0 +1,166 @@
+(JUST AN EXAMPLE FROM NNI, NEED REFINE. JIAHANG)
```

Comment:
Based on last meeting's discussion, should we remove this part (complex implementation in NNI) for customized backend registration?